splice/static/js/controllers/authoring.js

```diff
@@ -105,7 +105,18 @@ angular.module('spliceApp').controller('authoringController', function($controll
         var deployed = data.deployed;
         var msg = '<ol>';
         for (var url of data.urls) {
-          msg += '<li><a href="' + url + '">' + url + '</a></li>';
+
+          var uploadStatus = null;
+          var uploadClass = null;
+          if (url[1] == true) {
+            uploadStatus = "new";
+            uploadClass = "text-success";
+          }
+          else {
+            uploadStatus = "cached";
+            uploadClass = "text-muted";
```

Comment:
nit: you could move these two lines up to replace line 109/110 where you set them to null to save a couple of lines of code